features/support/bot_steps.ts

```diff
@@ -77,7 +77,7 @@ Then(
   /the bot does not respond with {(.*)}/,
   function (this: CustomWorld, expectedMessage: string) {
     const { bot } = this.mocks;
-    const calledWith = bot.say.lastArg;
+    const calledWith = bot.say.getCall(0).args;
```

Comment:
I'm not necessarily surprised you had to do this, but very surprised it didn't break anything else

Reply:
Yeah, the "does not respond" scenario wasn't working so I just copied what was working in the "does respond" definition.